### ISSUE TITLE
Website In Other Languages

### ISSUE THREAD

**Issue by ventolinmono** [keepassxreboot/keepassxreboot.github.io#78]
Hi, 

I would like to know if there are plans to make the website available in **Transifex**.
I give digital security workshops and some people find confusing the downloads section.
I could help translating some strings to **spanish**.

:)

**Comment by iamtheRichie** [comment]
I would find other languages also good 

I could contribute german ;)

**Comment by phoerious** [comment]
Generally yes. But the whole thing needs a redesign first.

**Comment by alsahawneh** [comment]
Me too!! I'm interested in translating the website into Arabic and / or contributing to the design (in terms of RTL).

**Comment by ventolinmono** [comment]
Yes, I can contribute in whatever platform the developer prefer.
